sources/Algo/include/Algo.h

```diff
@@ -230,22 +230,23 @@ class LoadDefinitionAlgorithm : public NodeAlgorithm {
    * @brief Constructor
    *
    * @param loads the list of loads to update
+   * @param dsoVoltageLevel the upper bound for the change of load model 
```

Comment:
This is a lower bound. I would propose "Minimum voltage level of the load to be taken into account".